### PR TITLE
fixed ValidationException

### PR DESCRIPTION
**Pull request by zagovorichev**
Pull request summary
 
Related to: https://oat-sa.atlassian.net/browse/TAO-8584
 
Fixed ValidationException (input parameter could be a file)
 
#### Steps to reproduce  (for bugs)
 - try to import broken qti (with incompatible xml, it could be manifest file and new or bad tag try `<mn/>`)
  
#### How to test
 
- Add to any qti package in the manifest file an empty tag `<mn/>`
- From the back office run import of the qti package

### CODE REVIEW

File: model/qti/parser/ValidationException.php [hunk 1]
```diff
@@ -35,20 +37,30 @@ public function __construct($file, $errors)
     {
         $this->errors = $errors;
         $this->xmlFile  = $file;
-        parent::__construct('Failed to validate '.$file);
+
+        parent::__construct('Failed to validate '.$this->getFilename());
     }
-    
+
+    private function getFilename()
```
Comment (yuryfilippovich) [inline model/qti/parser/ValidationException.php]: phpdoc

File: model/qti/parser/ValidationException.php [hunk 2]
```diff
@@ -35,20 +37,30 @@ public function __construct($file, $errors)
...
+    }
+
     /**
      * @return common_report_Report
      */
     public function getReport()
     {
-        return common_report_Report::createFailure(__("Malformed XML[%s]:\n%s", $this->xmlFile, implode("\n", $this->errors)));
+        return common_report_Report::createFailure(__("Malformed XML[%s]:\n%s", $this->getFilename(), implode("\n", $this->errors)));
     }
```
Comment (yuryfilippovich) [inline model/qti/parser/ValidationException.php]: phpdoc

File: model/qti/parser/ValidationException.php [hunk 3]
```diff
@@ -35,20 +37,30 @@ public function __construct($file, $errors)
     {
```
Comment (yuryfilippovich) [inline model/qti/parser/ValidationException.php]: * @param string|File $file in phpdoc

File: model/qti/parser/ValidationException.php [hunk 4]
```diff
@@ -35,20 +37,30 @@ public function __construct($file, $errors)
...
         $this->errors = $errors;
         $this->xmlFile  = $file;
-        parent::__construct('Failed to validate '.$file);
+
+        parent::__construct('Failed to validate '.$this->getFilename());
     }
-    
+
```
Comment (yuryfilippovich) [inline model/qti/parser/ValidationException.php]: phpdoc

**Comment by zagovorichev** [comment]
@yuryfilippovich Update

**Pull request merged**